modal/modal.html

```diff
@@ -27,7 +27,7 @@
                                 </button>
 
                                 <button class="col share clearfix" id="reddit" data-share="https://reddit.com/submit?url&amp;title">
-                                        <img src="icons/reddit.svg" />
+                                        <img src="icons/reddit.png" />
```

Comment:
I prefer to use SVG that avoid issues on rendering on different screens, when I found svg I use that format instead of others

Reply:
I don't find any new icon in svg format